### PR TITLE
Add calendar

### PR DESCRIPTION
**Pull request by RoeiRom**
#3

### CODE REVIEW

File: pages/calendar.vue [hunk 1]
```diff
@@ -4,43 +4,86 @@
...
 <script lang="ts">
 import Vue from 'vue';
 import CategoriesBar from '@/components/CalendarComponents/categoriesBar.vue';
-import { allEvents } from '@/queries/Events';
+import { AllEvents } from '@/queries/Events';
 import Event from '@/models/Event.ts';
 
+interface CalendarEvent {
+    name: string; 
+    start: string;
```
Comment (OrelKaravani) [inline pages/calendar.vue:31]: Why isn't a Date type?

File: pages/calendar.vue [hunk 2]
```diff
@@ -4,43 +4,86 @@
...
+    end: string;
```
Comment (OrelKaravani) [inline pages/calendar.vue:32]: Same as above.

File: pages/calendar.vue (hunk 1 above)
Comment (RoeiRom) [reply pages/calendar.vue:31]: To format to the calendar time time stamp to display,
YYYY-MM-DD HH:MM

Comment (OrelKaravani) [reply pages/calendar.vue:31]: Got it. You can do it with Date type aswell and for display use toString, but it's ok.

File: pages/calendar.vue [hunk 3]
```diff
@@ -4,43 +4,86 @@
...
+    color: string;
+};
+
 export default Vue.extend({
+    data() {
+        return {
+            start: ''
```
Comment (OrelKaravani) [inline pages/calendar.vue]: Give more meaningful name for that variable

File: pages/calendar.vue [hunk 4]
```diff
@@ -4,43 +4,86 @@
...
+                });
+            } else {
+                return [];
+            }
+        }, 
+        calendarEvents(): CalendarEvent[] {
+            return this.events.map((event: Event) => {
+                return {
+                    name: event.title, 
+                    start: this.formatDate(new Date(event.startDate)),
```
Comment (OrelKaravani) [inline pages/calendar.vue:74]: Use "startDate" as  event variable

File: pages/calendar.vue [hunk 5]
```diff
@@ -4,43 +4,86 @@
...
+                    end: this.formatDate(new Date(event.endDate)),
```
Comment (OrelKaravani) [inline pages/calendar.vue:75]: Same as above.

File: pages/calendar.vue (hunk 4 above)
Comment (RoeiRom) [reply pages/calendar.vue:74]: Calendar requires start and not startDate

File: pages/calendar.vue (hunk 5 above)
Comment (RoeiRom) [reply pages/calendar.vue:75]: Same as start

**Pull request merged**